projects/rawkode-academy/technologies-service/dagger/package.json

```diff
@@ -0,0 +1,7 @@
+{
+	"dependencies": {
+		"@dagger.io/dagger": "./sdk",
+		"typescript": "^5.5.4"
+	},
+	"packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
you could go higher by changing it to `yarn@3.8.5` as `1.22.22` is fairly old

Reply:
That's supposed to be deleted 😓